compose/README.md

```diff
@@ -0,0 +1 @@
+This folder contains several docker compose files for development.
```

Comment:
This folder should be added to `.docker-ignore`

Reply:
fixed: https://github.com/razee-io/Razeedash-api/blob/graphql/.dockerignore#L7